### PR TITLE
[IOTDB-4174] Show RegionNum when show storagegroup

### PR DESCRIPTION
**Pull request by CRZbulabula**
Recommend by @ericpai

Local test result:

image

I also rename cluster tools related cloumn constants in ColumnHeaderConstant by upper camel case by the way.

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java [hunk 1]
```diff
@@ -202,7 +202,7 @@ public StorageGroupSchemaResp getMatchedStorageGroupSchemas(GetStorageGroupPlan
       LOGGER.error("Error StorageGroup name", e);
       result.setStatus(
           new TSStatus(TSStatusCode.STORAGE_GROUP_NOT_EXIST.getStatusCode())
-              .setMessage("Error StorageGroup name"));
+              .setMessage("Error StorageGroup name: " + e));
```
Comment (ericpai) [inline confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java:205]: `+ e.getMessage()`? I think `getMessage()` is more friendly for human read.

File: confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java [hunk 2]
```diff
@@ -568,9 +567,13 @@ public TShowConfigNodesResp showConfigNodes() throws TException {
...
 
-  public void handleClientExit() {}
+  @Override
+  public TShowStorageGroupResp showStorageGroup(List<String> storageGroupPathPattern)
+      throws TException {
+    return configManager.showStorageGroup(new GetStorageGroupPlan(storageGroupPathPattern));
+  }
 
-  // TODO: Interfaces for data operations
+  public void handleClientExit() {}
```
Comment (ericpai) [inline confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java:576]: What's this empty method used for?

**Review by ericpai — CHANGES_REQUESTED** [review]

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java (hunk 1 above)
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java]: Fixed~

File: confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java (hunk 2 above)
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java]: This interface was added in March and now it's useless.

**Review by Beyyes — COMMENTED** [review]
Please take a look of CI, most tests run failed~

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/IManager.java [hunk 3]
```diff
@@ -306,6 +307,9 @@ Object getOrCreateDataPartition(
   /** Show ConfigNodes */
   TShowConfigNodesResp showConfigNodes();
 
+  /** Show StorageGroup */
+  TShowStorageGroupResp showStorageGroup(GetStorageGroupPlan getStorageGroupPlan);
```
[the remark this answers is not in the document]
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/IManager.java:316]: Fixed.

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java [hunk 4]
```diff
@@ -157,7 +157,7 @@ public TSStatus deleteStorageGroup(DeleteStorageGroupPlan plan) {
       LOGGER.warn("Storage group not exist", e);
       result
           .setCode(TSStatusCode.SUCCESS_STATUS.getStatusCode())
-          .setMessage("Storage group not exist");
+          .setMessage("Storage group not exist" + e.getMessage());
```
[the remark this answers is not in the document]
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java]: Fixed.

File: confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java [hunk 5]
```diff
@@ -176,7 +176,7 @@ public CountStorageGroupResp countMatchedStorageGroups(CountStorageGroupPlan pla
       LOGGER.error("Error StorageGroup name", e);
       result.setStatus(
           new TSStatus(TSStatusCode.STORAGE_GROUP_NOT_EXIST.getStatusCode())
-              .setMessage("Error StorageGroup name"));
+              .setMessage("Error StorageGroup name" + e.getMessage()));
```
[the remark this answers is not in the document]
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java]: Fixed.

[the remark this answers is not in the document]
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/persistence/schema/ClusterSchemaInfo.java]: Fixed.

**Review by ericpai — APPROVED** [review]

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**